src/core/Studio.ts

```diff
@@ -31,6 +52,114 @@ class Studio {
 
     return response;
   }
+
+  /**
+   * Uploads a video to YouTube.
+   * @example
+   * ```ts
+   * const buffer = fs.readFileSync('./my_awesome_video.mp4');
+   * const response = await yt.studio.upload(buffer, { title: 'Wow!' });
```

Comment:
Should this not be 
```ts
const response = await yt.studio.upload(new Uint8Array(buffer.buffer), { title: 'Wow!' });
```
since the type of the parameter is Uint8Array and not a Node.js Buffer?

Reply:
> Should this not be
> 
> ```ts
> const response = await yt.studio.upload(new Uint8Array(buffer.buffer), { title: 'Wow!' });
> ```
> 
> since the type of the parameter is Uint8Array and not a Node.js Buffer?

Doesn't `Buffer` extend JavaScript's `Uint8Array`? From my understanding TypeScript doesn't, and shouldn't complain about it.